tools/packaging/scripts/lib.sh

```diff
@@ -97,3 +97,19 @@ get_kata_hash() {
 	ref=$2
 	git ls-remote --heads --tags "https://github.com/${project}/${repo}.git" | grep "${ref}" | awk '{print $1}'
 }
+
+get_config_and_patches() {
```

Comment:
Those functions in `lib.sh` will be shared with other scripts? If yes, we need to document because here they loose context. Would make sense to pass `default_patches_dir` as parameter rather than assume it was exported?